### PR TITLE
Improve grammar in English README

### PR DESCRIPTION
**Pull request by navarr**
👍

### CODE REVIEW

**Comment by navarr** [comment]
READMEの英語はいいんけど、その方がいいんかも

すみません、私の拙い日本語

File: README.en.md [hunk 1]
```diff
@@ -16,59 +16,59 @@
...
 ## An Ideal Music Player
 
-A simple design of just playing the mp3 files under selected directory.
+Toa has a simple design, built to streamline it's purpose: playing MP3s from a directory.
 
-Toa has just enough UI to enjoy music, so Toa won't occupy your screen.
+Toa has just enough UI to enjoy music, so it won't take up unnecessary screen space.
 
-It also has the beautiful exterior of the flat design like below:
+It utilizes flat design, like so:
```
Comment (ghost) [inline README.en.md]: Thank you, but I'd rather prefer "as below" to "like so".

Comment (navarr) [reply README.en.md]: I thought it might be better to remove directionality.  Either way: changed!

**Comment by yui540** [comment]
Thank you!!!!

**Comment by ghost** [comment]
Thank you for improving my README 👍, only thing I'd rather prefer is explained as above, and The rest of English you've posted is definitely fine, I think.

これで殆ど改良する所はないと思います。

**Comment by yui540** [comment]
ありがとうございます！
お世話になりました！

**Pull request merged**